apps/settings/utils/ldap.py

```diff
@@ -665,7 +665,7 @@ def test_config(self):
     # test login
 
     def _test_before_login_check(self, username, password):
-        from settings.ws import CACHE_KEY_LDAP_TEST_CONFIG_TASK_STATUS, TASK_STATUS_IS_OVER
+        from settings.ws import CACHE_KEY_LDAP_TEST_CONFIG_TASK_STATUS
         if not cache.get(CACHE_KEY_LDAP_TEST_CONFIG_TASK_STATUS):
             self.test_config()
 
```

Comment:
I have reviewed the given code and found no immediate issues or inconsistencies that need to be addressed. The changes made seem to align with current conventions used for managing AD groups in Django models and views.

If you notice any further discrepancies or improvements necessary, please provide more details so I can assist you accordingly.